packages/forms-web-app/src/views/full-appeal/submit-final-comment/appeal-closed-for-comment.njk

```diff
@@ -0,0 +1,17 @@
+{# todo: change layout based on appeal type when component reused #}
+{% extends "layouts/full-appeal-banner/main.njk" %}
+
+{% block pageTitle %}
+  Appeal closed for comments - Appeal a planning decision - GOV.UK
```

Comment:
I think this should be `Appeal closed for comment - Submit final comment - Appeal a planning decision - GOV.UK`